emscripten.py

```diff
@@ -679,7 +679,7 @@ def make_export_wrappers(exports, delay_assignment):
     # The emscripten stack functions are called very early (by writeStackCookie) before
     # the runtime is initialized so we can't create these wrappers that check for
     # runtimeInitialized.
-    if settings.ASSERTIONS and not name.startswith('emscripten_stack_'):
+    if settings.ASSERTIONS and not name.startswith('emscripten_stack_') and name != '__trap':
```

Comment:
What is `__trap`? A comment here would be good to add to the existing comment about stack functions.

Reply:
Added comment.